terraform/dynamo.tf.json.template.py

```diff
@@ -77,6 +77,22 @@
                                 "type": "S"
                             }
                         ]
+                    },
+                    "failed_messages_table": {
```

Comment:
```suggestion
                    "failures_table": {
```

We should be more consistent. There is a mix of "failure", "failed", "failures".